content/docs/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Documentation"
+aliases:
+  - /csharp/
+  - /csharp/metrics/
+  - /csharp/tracing/
+  - /golang/
+  - /golang/metrics/
+  - /golang/tracing/
+  - /java/
```

Comment:
Why have `/java` link to `/docs` and not `/docs/java`? Aren't these aliases in the wrong place?